projects/tomlkit/atheris_tomlkit_dict.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# This is a modified version of https://github.com/robtandy/randomdict which has
+# been changed to
+#    - Work with the atheris fuzzer
+#    - Filter duplicate keys
+
+import datetime
+from itertools import product
+from random import choice, shuffle
```

Comment:
Does the use of `random` introduce non-determinism into the fuzzer? We should avoid non-determinism in fuzzers due to reproducible and the algorithms used by the fuzzer for coverage-feedback

Reply:
It is introducing non-determinism. I've replace the use of `choice` and `shuffle` with atheris generated data instead.